src/test/kotlin/uk/gov/justice/digital/hmpps/manageusersapi/resource/GroupsControllerIntTest.kt

```diff
@@ -578,4 +576,58 @@ class GroupsControllerIntTest : IntegrationTestBase() {
         .expectStatus().isUnauthorized
     }
   }
+
+  @Nested
+  inner class DeleteChildGroup {
+
+    @Test
+    fun `access forbidden when no authority`() {
+      webTestClient.delete().uri("/groups/child/CHILD_3")
+        .exchange()
+        .expectStatus().isUnauthorized
+    }
+
+    @Test
+    fun `access forbidden when no role`() {
+      webTestClient.delete().uri("/groups/child/CHILD_3")
+        .headers(setAuthorisation("ITAG_USER_ADM", listOf()))
+        .exchange()
+        .expectStatus().isForbidden
+    }
+
+    @Test
+    fun `access forbidden when wrong role`() {
+      webTestClient.delete().uri("/groups/child/CHILD_3")
+        .headers(setAuthorisation("ITAG_USER_ADM", listOf("ROLE_AUDIT")))
+        .exchange()
+        .expectStatus().isForbidden
+    }
+
+    @Test
+    fun `child group not found`() {
+      externalUsersApiMockServer.stubDeleteChildGroupFail("Not_A_Group", NOT_FOUND)
+      webTestClient
+        .delete().uri("/groups/child/Not_A_Group")
+        .headers(setAuthorisation("ITAG_USER_ADM", listOf("ROLE_MAINTAIN_OAUTH_USERS")))
+        .exchange()
+        .expectStatus().isNotFound
+        .expectHeader().contentType(MediaType.APPLICATION_JSON)
```

Comment:
Minor - can pull in APPLICATION_JSON rather than MediaType as import

Reply:
Done